src/containers/App/App.js

```diff
@@ -147,9 +147,8 @@ export /* istanbul ignore next */ class App extends Component {
     }
 
     const { loadingConfig } = this.state;
-    const { loadingConfig: prevLoadingConfig } = prevState;
 
-    if (prevLoadingConfig && loadingConfig === false) {
+    if (loadingConfig === false) {
```

Comment:
This causes namespaces and extensions to be requested repeatedly.

We could:
- undo these changes and continue using the existing `loadingConfig` in `componentDidUpdate` and `fetchConfig`
- add an additional flag `showLoadingState: true` to the initial state, set it to `false` after `fetchInstallProperties`
- use this to control the loading UI in `render`

Reply:
Hmm.. let's do that

Reply:
done